src/python.cc

```diff
@@ -1063,7 +1065,7 @@ TRITONBACKEND_Initialize(TRITONBACKEND_Backend* backend)
   triton::common::TritonJson::Value cmdline;
   backend_state->shm_default_byte_size = 64 * 1024 * 1024;  // 64 MBs
   backend_state->shm_growth_byte_size = 64 * 1024 * 1024;   // 64 MBs
-  backend_state->stub_timeout_seconds = 10;
+  backend_state->stub_timeout_seconds = 30;
```

Comment:
If you are changing the default value of stub timeout, you should update the docs as well. However what is the motivation to increase the default timeout if you are anyway increasing it to 120 in the L0_infer test